app/jobs/google_scraping_job.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require 'httparty'
+require 'nokogiri'
+
+class GoogleScrapingJob < ApplicationJob
+  retry_on Exception do |job, error|
+    keyword_id = job.arguments[0]
+
+    Keyword.update(
+      keyword_id,
+      status: :failed,
+      failed_reason: error
+    )
+  end
+
+  def perform(keyword_id, keyword)
+    scrap_from_google(keyword)
+    store_result(keyword_id)
```

Comment:
Should we separate this logic to a new class, e.g. a new `GoogleSrappingService` and call the service here. 

```ruby
Class GoogleScrapingService
   attr_reader :keyword_id, :keyword

   def initialize(keyword_id, keyword)
     @keyword_id = keyword_id
     @keyword = keyword
   end

   def call!
     scrap_from_google(keyword)
     store_result(keyword_id)
   end
end

class GoogleScrapingJob < ApplicationJob
   ..
   def perform(keyword_id, keyword)
      GoogleScrapingService.new(keyword_id, keyword).call!
   end
end
```

Reply:
Fixed in d6c74900c1fb2c53506402746039b448e86b4bd8